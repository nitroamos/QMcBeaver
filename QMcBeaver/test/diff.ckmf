# This will test diffusion, it should branch and fuse a lot of walkers

&flags
# Parameters for QMC
run_type
 diffusion
dt
 0.001
number_of_walkers
 10
max_time_steps
 200
max_time
 -1
one_e_per_iter
 0
desired_convergence
 0
warn_verbosity
 0
rel_cutoff
 100
limit_branching
 1

# Parameters specific to the Green's function
sampling_method
 umrigar93_importance_sampling
QF_modification_type
 umrigar93_unequalelectrons
umrigar93_equalelectrons_parameter
 0.5
energy_modification_type
 umrigar93
energy_cutoff_type
 umrigar93
lock_trial_energy
 0
synchronize_dmc_ensemble
 0
synchronize_dmc_ensemble_interval
 1000

# Parameters specific to weights, branching, and fusion
walker_reweighting_method
 umrigar93_probability_weighted
branching_method
 nonunit_weight_branching
branching_threshold
 1.01
fusion_threshold
 0.99
population_control_parameter
 1
correct_population_size_bias
 1
old_walker_acceptance_parameter
 30

# Parameters for initialization
dt_equilibration
 0.01
use_equilibration_array
 0
equilibration_steps
 100
equilibration_function
 ramp
CKAnnealingEquilibration1_parameter
 500
walker_initialization_method
 dans_walker_initialization
walker_initialization_combinations
 3

# Parameters for added functionality/improvements
calculate_bf_density
 0
use_hf_potential
 0
hf_num_average
 100
replace_electron_nucleus_cusps
 1
nuclear_derivatives
 none
future_walking
 0

# Parameters relating to output
output_interval
 5
checkpoint
 0
checkpoint_interval
 100000
use_available_checkpoints
 0
checkpoint_input_name
 gss4h_3
zero_out_checkpoint_statistics
 1
checkpoint_energy_only
 0
print_configs
 0
print_config_frequency
 50
temp_dir
 /temp2/amosa/
write_all_energies_out
 0
write_electron_densities
 0
max_pair_distance
 -1
print_transient_properties
 0
print_transient_properties_interval
 10000
writePllxCorrelationDiagram
 0
pllxCorrelationDiagramMin
 0
pllxCorrelationDiagramMax
 0
writePllyCorrelationDiagram
 0
pllyCorrelationDiagramMin
 0
pllyCorrelationDiagramMax
 0
writePllzCorrelationDiagram
 0
pllzCorrelationDiagramMin
 0
pllzCorrelationDiagramMax
 0
writeOppxCorrelationDiagram
 0
oppxCorrelationDiagramMin
 0
oppxCorrelationDiagramMax
 0
writeOppyCorrelationDiagram
 0
oppyCorrelationDiagramMin
 0
oppyCorrelationDiagramMax
 0
writeOppzCorrelationDiagram
 0
oppzCorrelationDiagramMin
 0
oppzCorrelationDiagramMax
 0

# Parameters for wavefunction optimization
optimize_Psi
 0
optimize_EE_Jastrows
 1
optimize_EN_Jastrows
 1
optimize_NEE_Jastrows
 1
optimize_L
 0
optimize_CI
 0
optimize_Orbitals
 0
optimize_Psi_method
 automatic
optimize_Psi_criteria
 generalized_eigenvector
a_diag
 -1e-05
ksi
 0.5
max_optimize_Psi_steps
 30
equilibrate_first_opt_step
 1
equilibrate_every_opt_step
 1
optimization_max_iterations
 1
optimization_error_tolerance
 0.001
singularity_penalty_function_parameter
 1e-06
optimize_Psi_barrier_parameter
 1
numerical_derivative_surface
 umrigar88
line_search_step_length
 Linearize
ck_genetic_algorithm_1_population_size
 1000
ck_genetic_algorithm_1_mutation_rate
 0.2
ck_genetic_algorithm_1_initial_distribution_deviation
 1

# Parameters for computation/MPI
iseed
 -1000
parallelization_method
 manager_worker
mpireduce_interval
 100
mpipoll_interval
 5
walkers_per_pass
 1
use_basis_function_interpolation
 0
number_basis_function_interpolation_grid_points
 1000
basis_function_interpolation_first_point
 1e-10

# Parameters for the molecule and wavefunction
atoms
 6
charge
 0
energy
 -78.0544263284
trial_function_type
 restricted
norbitals
 8
nbasisfunc
 62
ndeterminants
 1
link_Jastrow_parameters
 1
link_NEE_Jastrows
 2
link_Orbital_parameters
 1
link_Determinant_parameters
 1
use_three_body_jastrow
 1
reproduce_NE_with_NEE_jastrow
 1
reproduce_EE_with_NEE_jastrow
 1

# Other parameters
chip_and_mike_are_cool
 Yea_Baby!
& 
&geometry
C    6      0                     0                     1.257656            
C    6      0                     0                    -1.257656            
H    1     -1.743778              0                     2.320144            
H    1      1.743778              0                    -2.320144            
H    1     -1.743778              0                    -2.320144            
H    1      1.743778              0                     2.320144            
&
&basis
C     19    6
  6 s    
             4563.24    0.778201756489      
             682.024    1.44869480537       
             154.973    2.38308933222       
             44.4553    3.20008901227       
              13.029    3.01300443764       
             1.82773    0.247598632588      
  3 s    
             20.9642    0.800628519338      
             4.80331    2.12741989144       
             1.45933   -0.00286791063143    
  3 px   
             20.9642    2.57359221527       
             4.80331    2.40824822281       
             1.45933    1.86528133353       
  3 py   
             20.9642    2.57359221527       
             4.80331    2.40824822281       
             1.45933    1.86528133353       
  3 pz   
             20.9642    2.57359221527       
             4.80331    2.40824822281       
             1.45933    1.86528133353       
  1 s    
            0.483456    0.413216645796      
  1 px   
            0.483456    0.574627334294      
  1 py   
            0.483456    0.574627334294      
  1 pz   
            0.483456    0.574627334294      
  1 s    
            0.145585    0.167976127675      
  1 px   
            0.145585    0.128184607047      
  1 py   
            0.145585    0.128184607047      
  1 pz   
            0.145585    0.128184607047      
  1 dxx  
               0.626    0.725127773044      
  1 dyy  
               0.626    0.725127773044      
  1 dzz  
               0.626    0.725127773044      
  1 dxy  
               0.626    1.25595814489       
  1 dxz  
               0.626    1.25595814489       
  1 dyz  
               0.626    1.25595814489       
C     19    6
  6 s    
             4563.24    0.778201756489      
             682.024    1.44869480537       
             154.973    2.38308933222       
             44.4553    3.20008901227       
              13.029    3.01300443764       
             1.82773    0.247598632588      
  3 s    
             20.9642    0.800628519338      
             4.80331    2.12741989144       
             1.45933   -0.00286791063143    
  3 px   
             20.9642    2.57359221527       
             4.80331    2.40824822281       
             1.45933    1.86528133353       
  3 py   
             20.9642    2.57359221527       
             4.80331    2.40824822281       
             1.45933    1.86528133353       
  3 pz   
             20.9642    2.57359221527       
             4.80331    2.40824822281       
             1.45933    1.86528133353       
  1 s    
            0.483456    0.413216645796      
  1 px   
            0.483456    0.574627334294      
  1 py   
            0.483456    0.574627334294      
  1 pz   
            0.483456    0.574627334294      
  1 s    
            0.145585    0.167976127675      
  1 px   
            0.145585    0.128184607047      
  1 py   
            0.145585    0.128184607047      
  1 pz   
            0.145585    0.128184607047      
  1 dxx  
               0.626    0.725127773044      
  1 dyy  
               0.626    0.725127773044      
  1 dzz  
               0.626    0.725127773044      
  1 dxy  
               0.626    1.25595814489       
  1 dxz  
               0.626    1.25595814489       
  1 dyz  
               0.626    1.25595814489       
H      6    3
  3 s    
              33.865    0.255069319975      
             5.09479    0.460108662165      
             1.15879    0.678321446492      
  1 s    
             0.32584    0.307371353862      
  1 s    
            0.102741    0.129335555758      
  1 px   
                0.75    0.994870760731      
  1 py   
                0.75    0.994870760731      
  1 pz   
                0.75    0.994870760731      
H      6    3
  3 s    
              33.865    0.255069319975      
             5.09479    0.460108662165      
             1.15879    0.678321446492      
  1 s    
             0.32584    0.307371353862      
  1 s    
            0.102741    0.129335555758      
  1 px   
                0.75    0.994870760731      
  1 py   
                0.75    0.994870760731      
  1 pz   
                0.75    0.994870760731      
H      6    3
  3 s    
              33.865    0.255069319975      
             5.09479    0.460108662165      
             1.15879    0.678321446492      
  1 s    
             0.32584    0.307371353862      
  1 s    
            0.102741    0.129335555758      
  1 px   
                0.75    0.994870760731      
  1 py   
                0.75    0.994870760731      
  1 pz   
                0.75    0.994870760731      
H      6    3
  3 s    
              33.865    0.255069319975      
             5.09479    0.460108662165      
             1.15879    0.678321446492      
  1 s    
             0.32584    0.307371353862      
  1 s    
            0.102741    0.129335555758      
  1 px   
                0.75    0.994870760731      
  1 py   
                0.75    0.994870760731      
  1 pz   
                0.75    0.994870760731      
&
&wavefunction
  0.398310267                0.329419253                0                        
  0                          0.000114734319             0.00361177591            
  0                          0                         -0.000784766947           
  0.000722206981             0                          0                        
  6.27604651e-05            -0.00111461511             -0.00086801121            
 -0.000425232794             0                          0                        
  0                          0.398310267                0.329419253              
  0                          0                         -0.000114734319           
  0.00361177591              0                          0                        
  0.000784766947             0.000722206981             0                        
  0                         -6.27604651e-05            -0.00111461511            
 -0.00086801121             -0.000425232794             0                        
  0                          0                          0.000350006871           
  0.00042263081             -0.000198145207             0.000284828832           
  0                         -0.00019267274              0.000350006871           
  0.00042263081             -0.000198145207            -0.000284828832           
  0                          0.00019267274              0.000350006871           
  0.00042263081             -0.000198145207             0.000284828832           
  0                          0.00019267274              0.000350006871           
  0.00042263081             -0.000198145207            -0.000284828832           
  0                         -0.00019267274            

  0.398476109                0.329710365                0                        
  0                          0.000348205729             0.00301521658            
  0                          0                          0.000124782243           
  0.00141139258              0                          0                        
 -0.000331315927            -0.000717755715            -0.000865458964           
 -0.00087560423              0                          0                        
  0                         -0.398476109               -0.329710365              
  0                          0                          0.000348205729           
 -0.00301521658              0                          0                        
  0.000124782243            -0.00141139258              0                        
  0                         -0.000331315927             0.000717755715           
  0.000865458964             0.00087560423              0                        
  0                          0                          0.000294181974           
  0.000305629488            -0.000125945901             0.000224772161           
  0                         -0.000110156188            -0.000294181974           
 -0.000305629488             0.000125945901             0.000224772161           
  0                         -0.000110156188            -0.000294181974           
 -0.000305629488             0.000125945901            -0.000224772161           
  0                         -0.000110156188             0.000294181974           
  0.000305629488            -0.000125945901            -0.000224772161           
  0                         -0.000110156188           

 -0.0836912145              -0.137163811                0                        
  0                         -0.0500372849               0.346107872              
  0                          0                         -0.0768645363             
  0.182991919                0                          0                        
 -0.0137096417              -0.00424906004             -0.0105150826             
  0.0189088684               0                          0                        
  0                         -0.0836912145              -0.137163811              
  0                          0                          0.0500372849             
  0.346107872                0                          0                        
  0.0768645363               0.182991919                0                        
  0                          0.0137096417              -0.00424906004            
 -0.0105150826               0.0189088684               0                        
  0                          0                          0.05141433               
  0.0654521664              -0.00321449828              0.0103012787             
  0                         -0.00843134153              0.05141433               
  0.0654521664              -0.00321449828             -0.0103012787             
  0                          0.00843134153              0.05141433               
  0.0654521664              -0.00321449828              0.0103012787             
  0                          0.00843134153              0.05141433               
  0.0654521664              -0.00321449828             -0.0103012787             
  0                         -0.00843134153            

 -0.0638787099              -0.106434846                0                        
  0                          0.0708754591               0.271932144              
  0                          0                          0.116744607              
  0.207785987                0                          0                        
  0.0481488786               0.0163798317              -0.0095119367             
 -0.00806440065              0                          0                        
  0                          0.0638787099               0.106434846              
  0                          0                          0.0708754591             
 -0.271932144                0                          0                        
  0.116744607               -0.207785987                0                        
  0                          0.0481488786              -0.0163798317             
  0.0095119367               0.00806440065              0                        
  0                          0                          0.0881482431             
  0.125170289                0.0164612304               0.015228308              
  0                         -0.00534507488             -0.0881482431             
 -0.125170289               -0.0164612304               0.015228308              
  0                         -0.00534507488             -0.0881482431             
 -0.125170289               -0.0164612304              -0.015228308              
  0                         -0.00534507488              0.0881482431             
  0.125170289                0.0164612304              -0.015228308              
  0                         -0.00534507488            

  0                          0                          0.140704031              
  0                          0                          0                        
  0.214301913                0                          0                        
  0                          0.112218808                0                        
  0                          0                          0                        
  0                          0                          0.00415706115            
  0                          0                          0                        
  0.140704031                0                          0                        
  0                          0.214301913                0                        
  0                          0                          0.112218808              
  0                          0                          0                        
  0                          0                          0                        
 -0.00415706115              0                         -0.094416357              
 -0.144959665               -0.0380222637              -0.010812333              
  0                          0.0102693023               0.094416357              
  0.144959665                0.0380222637              -0.010812333              
  0                          0.0102693023              -0.094416357              
 -0.144959665               -0.0380222637              -0.010812333              
  0                         -0.0102693023               0.094416357              
  0.144959665                0.0380222637              -0.010812333              
  0                         -0.0102693023             

  0.0060010297               0.0102400263               0                        
  0                          0.183338651               -0.0310059196             
  0                          0                          0.274777922              
  0.0213840043               0                          0                        
  0.142032566                0.0230078619              -0.00157698907            
 -0.0199077841               0                          0                        
  0                          0.0060010297               0.0102400263             
  0                          0                         -0.183338651              
 -0.0310059196               0                          0                        
 -0.274777922                0.0213840043               0                        
  0                         -0.142032566                0.0230078619             
 -0.00157698907             -0.0199077841               0                        
  0                          0                          0.0745700088             
  0.117731846                0.0422298307               0.0123835745             
  0                          0.00270488529              0.0745700088             
  0.117731846                0.0422298307              -0.0123835745             
  0                         -0.00270488529              0.0745700088             
  0.117731846                0.0422298307               0.0123835745             
  0                         -0.00270488529              0.0745700088             
  0.117731846                0.0422298307              -0.0123835745             
  0                          0.00270488529            

  0                          0                          0.133526987              
  0                          0                          0                        
  0.20568443                 0                          0                        
  0                          0.158876598                0                        
  0                          0                          0                        
  0                          0                          0.0403582093             
  0                          0                          0                        
 -0.133526987                0                          0                        
  0                         -0.20568443                 0                        
  0                          0                         -0.158876598              
  0                          0                          0                        
  0                          0                          0                        
  0.0403582093               0                         -0.111375456              
 -0.17880996                -0.0814999347              -0.00675952921            
  0                          0.00916532381             -0.111375456              
 -0.17880996                -0.0814999347               0.00675952921            
  0                         -0.00916532381              0.111375456              
  0.17880996                 0.0814999347               0.00675952921            
  0                          0.00916532381              0.111375456              
  0.17880996                 0.0814999347              -0.00675952921            
  0                         -0.00916532381            

  0                          0                          0                        
  0.165334842                0                          0                        
  0                          0.269225892                0                        
  0                          0                          0.278825003              
  0                          0                          0                        
  0                          0                          0                        
 -0.0310454988               0                          0                        
  0                          0.165334842                0                        
  0                          0                          0.269225892              
  0                          0                          0                        
  0.278825003                0                          0                        
  0                          0                          0                        
  0                          0.0310454988               0                        
  0                          0                          0                        
  0.0125739212               0                          0                        
  0                          0                          0                        
  0.0125739212               0                          0                        
  0                          0                          0                        
  0.0125739212               0                          0                        
  0                          0                          0                        
  0.0125739212               0                        

Alpha Occupation
1   1   1   1   1   1   1   1   

Beta Occupation
1   1   1   1   1   1   1   1   

CI Coeffs
  1                        

&

&Jastrow
ParticleTypes:	Electron_up	Electron_down
CorrelationFunctionType:	Cambridge2
NumberOfParameterTypes:	2
NumberOfParametersOfEachType:	1	8
Parameters:	0.13229620434	0.840516507229	3.21625978344	-5.81673746473	16.6629247036	-2.55756976425	0.488191718116	-39.4811007227	37.6365725343
NumberOfConstantTypes:	2
NumberOfConstantsOfEachType:	1	1
Constants:	0.5	3

ParticleTypes:	Electron_up	Electron_up
CorrelationFunctionType:	Cambridge2
NumberOfParameterTypes:	2
NumberOfParametersOfEachType:	1	8
Parameters:	0.165051392496	0.567343072593	-4.97951764177	14.8734863417	-8.49390953645	-2.10689913182	-3.16203506241	-1.59768602689	17.7120212869
NumberOfConstantTypes:	2
NumberOfConstantsOfEachType:	1	1
Constants:	0.25	3

ParticleTypes:	Electron_up	C
CorrelationFunctionType:	Cambridge2
NumberOfParameterTypes:	2
NumberOfParametersOfEachType:	1	8
Parameters:	0.143206439691	-2.51121146009	10.6613415944	-9.43975002223	23.584665801	-71.9595355108	12.4725635808	44.6970912494	-25.2384561434
NumberOfConstantTypes:	2
NumberOfConstantsOfEachType:	1	1
Constants:	0	3

ParticleTypes:	Electron_up	H
CorrelationFunctionType:	Cambridge2
NumberOfParameterTypes:	2
NumberOfParametersOfEachType:	1	8
Parameters:	0.228584083379	-0.200708174897	-1.39399316422	6.3086557804	-26.9594977292	56.7287457813	-30.0314453296	-56.7139291916	64.1286198143
NumberOfConstantTypes:	2
NumberOfConstantsOfEachType:	1	1
Constants:	0	3

ParticleTypes:	C	Electron_up	Electron_down
threeBodyCorrelationFunctionType:	Cambridge
NumberOfParameterTypes:	2
NumberOfParametersOfEachType:  4  4
Parameters:
  0.091586015485      

  0.274758046455        0.274758046455        0.824274139365        0                   
  0                     0                     32.704150799        

  2.9251694486          2.9251694486         -39.4211140356        -39.4211140356       
  130.330440125        -16.3520753995        -16.3520753995        -3.27918201736       
 -3.27918201736         10.6950005752        -7.30025875433        -6.40830468052       
 -6.40830468052         4.15460778237         24.65097406           24.65097406         
 -9.84099123933        -9.84099123933         31.7364361831       

  6.42357631202         6.42357631202         37.5834247487         37.5834247487       
 -104.199670843        -104.199670843        -0.219740844543        3.27918201736       
  3.27918201736        -5.34750028761        -5.34750028761         0                   
  0                     0                    -7.02760978308        -7.02760978308       
 -43.9276617382        -43.9276617382         38.5240990578         38.5240990578       
  22.5819219446         21.1453292502        -24.033274067         -24.033274067        
  26.879137558         -6.29770890722        -6.29770890722         3.95170566728       
  3.95170566728        -14.3327069076         9.10836765945         9.10836765945       
  27.3251029784         27.3251029784        -44.3087950956        -44.3087950956       
 -9.09185241491       

C:  3
Cutoff:  0.262364368507


&Jastrow


