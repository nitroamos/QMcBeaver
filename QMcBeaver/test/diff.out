QMcBeaver version 20080225 was compiled in double precision.
Compiled on Feb 25 2008 23:08:28
Compiler version 3.4.6 20060404 (Red Hat 3.4.6-8)
With the libraries: MPI BLAS LAPACK
Signaling: CHANNEL1 = 23 CHANNEL2 = 40


EupEdn:
Cambridge 2 particle jastrow parameters:
  g = 0.5  C = 3  L = 0.132296 (not optimized)
x = r / 7.558796
(x - 1)^3 (0.8405165 -1.257848 x +3.21626 x^2 -5.816737 x^3 +16.66292 x^4 -2.55757 x^5 +0.4881917 x^6 -39.4811 x^7 +37.63657 x^8 )

EupEup:
Cambridge 2 particle jastrow parameters:
  g = 0.25  C = 3  L = 0.1650514 (not optimized)
x = r / 6.058719
(x - 1)^3 (0.5673431 +0.1873495 x -4.979518 x^2 +14.87349 x^3 -8.49391 x^4 -2.106899 x^5 -3.162035 x^6 -1.597686 x^7 +17.71202 x^8 )

EupNuclear(C):
Cambridge 2 particle jastrow parameters:
  g = 0  C = 3  L = 0.1432064 (not optimized)
x = r / 6.982926
(x - 1)^3 (-2.511211 -7.533634 x +10.66134 x^2 -9.43975 x^3 +23.58467 x^4 -71.95954 x^5 +12.47256 x^6 +44.69709 x^7 -25.23846 x^8 )

EupNuclear(H):
Cambridge 2 particle jastrow parameters:
  g = 0  C = 3  L = 0.2285841 (not optimized)
x = r / 4.374758
(x - 1)^3 (-0.2007082 -0.6021245 x -1.393993 x^2 +6.308656 x^3 -26.9595 x^4 +56.72875 x^5 -30.03145 x^6 -56.71393 x^7 +64.12862 x^8 )

EupEdnNuclear(C):
 Total Parameters = 64
 Free Parameters = 27
Cambridge 3 particle jastrow parameters:
  Nen = 4  Nee = 4  C = 3  L = 0.2623644 (not optimized)
r1  = x1  / 3.811493
r2  = x2  / 3.811493
r12 = x12 / 3.811493
(r1 - 1)^3 (r2 - 1)^3 (
+0.09158602                              +0.274758  (r1 + r2)                     +0.8242741 r1 r2                         +32.70415  r1 r2 r12                     
+2.925169  (r1^2 + r2^2)                 -39.42111  (r1^2 r2 + r2^2 r1)           +130.3304  r1^2 r2^2                     -16.35208  (r1^2 + r2^2) r12             
-3.279182  (r1^2 r2 + r2^2 r1) r12       +10.695    r1^2 r2^2 r12                 -7.300259  r12^2                         -6.408305  (r1 + r2) r12^2               
+4.154608  r1 r2 r12^2                   +24.65097  (r1^2 + r2^2) r12^2           -9.840991  (r1^2 r2 + r2^2 r1) r12^2     +31.73644  r1^2 r2^2 r12^2               
+6.423576  (r1^3 + r2^3)                 +37.58342  (r1^3 r2 + r2^3 r1)           -104.1997  (r1^3 r2^2 + r2^3 r1^2)       -0.2197408 r1^3 r2^3                     
+3.279182  (r1^3 + r2^3) r12             -5.3475    (r1^3 r2 + r2^3 r1) r12       -7.02761   (r1^3 + r2^3) r12^2           -43.92766  (r1^3 r2 + r2^3 r1) r12^2     
+38.5241   (r1^3 r2^2 + r2^3 r1^2) r12^2 +22.58192  r1^3 r2^3 r12^2               +21.14533  r12^3                         -24.03327  (r1 + r2) r12^3               
+26.87914  r1 r2 r12^3                   -6.297709  (r1^2 + r2^2) r12^3           +3.951706  (r1^2 r2 + r2^2 r1) r12^3     -14.33271  r1^2 r2^2 r12^3               
+9.108368  (r1^3 + r2^3) r12^3           +27.3251   (r1^3 r2 + r2^3 r1) r12^3     -44.3088   (r1^3 r2^2 + r2^3 r1^2) r12^3 -9.091852  r1^3 r2^3 r12^3               )

***************  TheMan.run();
 iteration            Eavg            Estd    Num. Walkers         Weights    Trial Energy         Eff. dt    Num. Samples            Skew        Kurtosis
       -95    0.0000000000   0.0000000e+00              10   10.0668697652  -78.8594373085   8.2790397e-03               0       0.0000000       0.0000000 
       -90    0.0000000000   0.0000000e+00              13   10.0509053145  -78.6942664536   7.9357613e-03               0       0.0000000       0.0000000 
       -85    0.0000000000   0.0000000e+00              11   10.0064746365  -78.1400763283   7.5569660e-03               0       0.0000000       0.0000000 
       -80    0.0000000000   0.0000000e+00              13   10.0339580330  -78.5263773895   7.1830553e-03               0       0.0000000       0.0000000 
       -75    0.0000000000   0.0000000e+00              11   10.1221093428  -79.8410283885   6.7933327e-03               0       0.0000000       0.0000000 
       -70    0.0000000000   0.0000000e+00              13   10.0476665906  -78.7990656796   6.3860908e-03               0       0.0000000       0.0000000 
       -65    0.0000000000   0.0000000e+00              11   10.0842121225  -79.4553026981   5.9862182e-03               0       0.0000000       0.0000000 
       -60    0.0000000000   0.0000000e+00              11    9.9241110470  -76.6931169421   5.5959634e-03               0       0.0000000       0.0000000 
       -55    0.0000000000   0.0000000e+00              10    9.9902877130  -77.8679604391   5.2111442e-03               0       0.0000000       0.0000000 
       -50    0.0000000000   0.0000000e+00              11   10.0141679841  -78.3477759962   4.8263075e-03               0       0.0000000       0.0000000 
       -45    0.0000000000   0.0000000e+00              10   10.0236131296  -78.5859645176   4.4371778e-03               0       0.0000000       0.0000000 
       -40    0.0000000000   0.0000000e+00              11   10.0247287814  -78.6648943836   4.0457901e-03               0       0.0000000       0.0000000 
       -35    0.0000000000   0.0000000e+00              10    9.9954455483  -77.9297406013   3.6535771e-03               0       0.0000000       0.0000000 
       -30    0.0000000000   0.0000000e+00              11    9.9962702348  -77.9400038534   3.2602519e-03               0       0.0000000       0.0000000 
       -25    0.0000000000   0.0000000e+00              10   10.0060634608  -78.2659279733   2.8659934e-03               0       0.0000000       0.0000000 
       -20    0.0000000000   0.0000000e+00              11   10.0145061494  -78.6410685825   2.4709502e-03               0       0.0000000       0.0000000 
       -15    0.0000000000   0.0000000e+00              10   10.0083097672  -78.4547684480   2.0748045e-03               0       0.0000000       0.0000000 
       -10    0.0000000000   0.0000000e+00              11   10.0122223391  -78.7825328359   1.6776221e-03               0       0.0000000       0.0000000 
        -5    0.0000000000   0.0000000e+00              10   10.0129045549  -79.0623226454   1.2795201e-03               0       0.0000000       0.0000000 
         0  -78.4666375214               0              11    9.9999995542  -78.0543757049   8.8055949e-04             100       0.3135542       0.9231905 
       105  -78.4489488754   8.4168442e-02              10   10.0027612683  -78.1160483350   9.9725628e-04             105       0.2316261       0.9055210 
       110  -78.4402040565               0              11   10.0041010786  -78.1917865674   9.9248593e-04             110       0.2032176       0.9305520 
       115  -78.4507684374   7.8838356e-02              10   10.0226120781  -78.3534932892   9.9213005e-04             115       0.2244539       0.9108239 
       120  -78.4937120026               0              11   10.0723106944  -78.6280638660   9.8350092e-04             120       0.0768939       0.7321268 
       125  -78.4704855835               0              10   10.0403734991  -78.4889460594   9.7852227e-04             125       0.1196322       0.4917924 
       130  -78.4840243944               0              11   10.0504037281  -78.5433254587   9.7708460e-04             130       0.1178982       0.3980562 
       135  -78.5482835068               0              10   10.1296418165  -78.7791656357   9.7883436e-04             135      -0.3461666       0.8438891 
       140  -78.5780652948   1.1499053e-01              11   10.1485832944  -78.8525054246   9.7803685e-04             140      -0.3041204       0.5878716 
       145  -78.6343112555               0              10   10.2033946767  -79.0008321930   9.7636289e-04             145      -0.3903590       0.3291071 
       150  -78.6793367593               0              11   10.2441650331  -79.0976510613   9.7691525e-04             150      -0.3580345       0.0576065 
       155  -78.7228641878               0              10   10.2905978621  -79.1816711346   9.7770464e-04             155      -0.3398787      -0.1273046 
       160  -78.7517355426               0              11   10.3097930769  -79.2193920122   9.7911138e-04             160      -0.2639998      -0.2523622 
       165  -78.7573723761               0              10   10.2964617154  -79.1966340311   9.7913888e-04             165      -0.2472483      -0.1925979 
       170  -78.7561967574               0              11   10.2738482808  -79.1611927427   9.7956224e-04             170      -0.2540781      -0.1209294 
       175  -78.7727444339               0              10   10.2786086755  -79.1720007604   9.8024412e-04             175      -0.2066659      -0.1542823 
       180  -78.7881089616               0              11   10.2887300568  -79.1814430516   9.8109439e-04             180      -0.1603532      -0.1607627 
       185  -78.7846029099               0              10   10.2644831388  -79.1493313540   9.8171896e-04             185      -0.1759251      -0.0863495 
       190  -78.7920415196               0              11   10.2593857882  -79.1439813033   9.8189882e-04             190      -0.1503413      -0.0415190 
       195  -78.7692907155               0              10   10.2073233799  -79.0765544076   9.8161349e-04             195      -0.1787386      -0.0891353 
       200  -79.0071538901               0              11   10.1533296732  -79.0223704954   9.8213946e-04             100       0.0709018      -0.6726054 
       200  -79.0071538901               0              11   10.1533296732  -79.0223704954   9.8213946e-04             100       0.0709018      -0.6726054 
***************  Print Results (ok = 1)
**************** Start of Results *****************

------------------- Energy -------------------
Sample variance = 0.5947417
 -7.900715389011e+01 +/-   9.900000000000e+01 (100 samples, decorr depth -1)
  DeCorr_depth       samples           Ave           Std        StdStd           Var        VarStd    Corr. Len.      Variance          Skew      Kurtosis
             0           100    -79.007154  7.750801e-02  5.508255e-03  6.007492e-03  8.538677e-04      1.000000      0.594742      0.070902     -0.672605
             1            50    -79.007154  1.055798e-01  1.066517e-02  1.114709e-02  2.252052e-03      1.855531      0.546207      0.084591     -0.687942
             2            25    -79.007154  1.404874e-01  2.027761e-02  1.973670e-02  5.697496e-03      3.285348      0.473681      0.059601     -1.077648
             3            12    -79.040695  1.712636e-01  3.651351e-02  2.933120e-02  1.250687e-02      4.882437      0.322643     -0.315004     -1.093070
             4             6    -79.040695  2.320227e-01  7.337201e-02  5.383453e-02  3.404794e-02      8.961231      0.269173     -0.233915     -1.373097
             5             3    -79.040695  3.292934e-01  1.646467e-01  1.084342e-01  1.084342e-01     18.049823      0.216868     -0.647347     -1.499192
             6             1    -79.173510

----------------- Energy^2 -------------------
  6.242725107531e+03 +/-   9.900000000000e+01 (100 samples, decorr depth -1)

--------------- Kinetic Energy ---------------
  8.379593684497e+01 +/-   5.351015824224e+00 (100 samples, decorr depth 3)

-------------- Potential Energy --------------
 -1.628030907351e+02 +/-   5.361767519261e+00 (100 samples, decorr depth 3)

-------------- Nuc-Elec Energy ---------------
 -2.536751896603e+02 +/-   5.527706169326e+00 (100 samples, decorr depth 3)

-------------- Elec-Elec Energy --------------
  5.742742741579e+01 +/-   4.506179202184e-01 (100 samples, decorr depth 3)

------------ AcceptanceProbability -----------
  9.841677663809e-01 +/-   9.900000000000e+01 (100 samples, decorr depth -1)

------------ DistanceMovedAccepted -----------
  4.694280324859e-02 +/-   3.608560085230e-04 (100 samples, decorr depth 1)

------------- DistanceMovedTrial -------------
  4.779647411399e-02 +/-   3.385629164181e-04 (100 samples, decorr depth 0)

----------------- Walker Age -----------------
  1.050000000000e+01 +/-   5.025189076296e-02 (100 samples, decorr depth 0)

--------------- Weight Change ----------------
  1.000188200042e+00 +/-   9.900000000000e+01 (100 samples, decorr depth -1)

---------------- Growth Rate -----------------
  0.000000000000e+00 +/-   1.005037815259e-01 (100 samples, decorr depth 0)

----------------- logWeights -----------------
  9.689513779639e-01 +/-   4.767360932264e-03 (100 samples, decorr depth 0)
dt_effective: 9.821394594221e-04 +/- 3.246518679029e-04

--------------- Virial Ratio (-<V>/<T>) ---------------
      0: 1.942851847773e+00 +/- 1.395943920940e-01
         Total Energy Estimator (KE/Virial): -8.379593684497e+01 +/- 5.351015824224e+00
         Total Energy Estimator (PE/Virial): -8.140154536754e+01 +/- 2.680883759631e+00

**************** End of Results *****************
***************  Finalize
Average iterations per hour:                              3.392895935641e+05
Average iterations*walkers per hour:                      3.392895935641e+06
Average microseconds per sample:                          9999.22
Average microseconds per sample per num initial walkers:  999.922

Wallclock Time:                         2122 ms   =                2.122 s    =             0.035367 mins =           0.00058944 hrs 
Total Time:                             2122 ms   =                2.122 s    =             0.035367 mins =           0.00058944 hrs     100.00 %
Initialization Time:                      10 ms   =                 0.01 s    =           0.00016667 mins =           2.7778e-06 hrs       0.49 %
Equilibration Time:                     1086 ms   =                1.086 s    =               0.0181 mins =           0.00030167 hrs      51.20 %
Propagation Time:                        999 ms   =                0.999 s    =              0.01665 mins =            0.0002775 hrs      47.12 %
Send Command Time:                         0 ms   =                    0 s    =                    0 mins =                    0 hrs       0.00 %
Gather Properties Time:                    0 ms   =                    0 s    =                    0 mins =                    0 hrs       0.01 %
Synchronization Time:                      0 ms   =                    0 s    =                    0 mins =                    0 hrs       0.00 %
Poll for Command Time:                     0 ms   =                    0 s    =                    0 mins =                    0 hrs       0.00 %
Optimization Time:                         0 ms   =                    0 s    =                    0 mins =                    0 hrs       0.00 %

Wallclock Time:                         2138 ms   =                2.138 s    =             0.035633 mins =           0.00059389 hrs 
